go.mod

```diff
@@ -2,18 +2,18 @@ module github.com/cyverse-de/search
 
 go 1.16
 
+replace github.com/cyverse-de/querydsl => ../querydsl
```

Comment:
This replacement will have to be removed before this can be merged.

Reply:
Yup, I figured I'd merge the querydsl part first and then update this!